Features/Context/PlatformUI.php

```diff
@@ -78,6 +79,7 @@ protected function waitForLoadings()
         while ($page->find('css', $loadingSelector) != null) {
             usleep(100 * 1000); // 100ms
         }
+        sleep(1);
```

Comment:
I decided to leave this here because this way it does not happen the "random" race condition errors, this is more of a temporary solution while we don't find a better/definitive one, but I forgot to had some comments about it @andrerom